app/services/reporters/event_service_reporter.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Reporters
+  # Reports to DOR Event Service.
+  class EventServiceReporter < BaseReporter
+    protected
+
+    def handled_single_codes
+      [
+        AuditResults::INVALID_MOAB
+      ]
+    end
+
+    def handled_merge_codes
+      [
+        AuditResults::ACTUAL_VERS_LT_DB_OBJ,
```

Comment:
honestly, I'm not sure these need to be in AuditResults namespace, but certainly wouldn't hold this up for that.